proto/tendermint/version/types.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
```

Comment:
Is this file needed? Or it's just side-effect of running `./proto/get_deps.sh`?

Reply:
it's just side-effect of running ./proto/get_deps.sh

Reply:
Removed it